docs/usage.rst

```diff
@@ -1,34 +1,38 @@
 Using ``pyramid_nacl_session``
 ==============================
 
-Generating a Random Secret
---------------------------
+Installation
+------------
 
-First, you need to arrange to have a 32-byte random secret available:
+Install using setuptools, e.g. (within a virtualenv)::
 
-.. code-block:: python
+  $ pip install pyramid_nacl_session
 
-   from pyramid_nacl_session import generate_secret
+Setup
+-----
 
-   SECRET = generate_secret(as_hex=False)
+Once :mod:`pyramid_nacl_session` is installed, you typically use the 
```

Comment:
When making changes to the docs, [please use one sentence per line](https://docs.pylonsproject.org/projects/docs-style-guide/#use-of-whitespace).